packages/web-components/src/select/select.spec.md

```diff
@@ -0,0 +1,92 @@
+# Select
+
+## Component Description
+
+Select is a list in which the selected item is always visible while other items are visible on demand by clicking a dropdown button.
```

Comment:
Select is a form associated control as well. It should not simply be used as a list IMO.